### ISSUE TITLE
Emit can't handle more than 1 parameter in views

### ISSUE THREAD

**Issue by YerkoPalma** [choojs/choo#614]
### Expected behavior

This code (view) should work
```js
function someView (state, emit) {
    return html`
      <body>        
        <p>Send something through sockets:</p>
        <input id="message" />
        <button onclick=${onclick}>Send</button>        
      </body>
    `

    function onclick () {
      emit('ws:send', document.getElementById('message').value, id)      
      document.getElementById('message').value = ''
    }
  }
```

### Actual behavior
The `id` parameter in the `ws:send` event is `undefined`

### Steps to reproduce behavior
Put that code in a view

---

This issue is only related to the way choo handle views, it is not related to nanobus. this can be checked running the same example in a store instead of a view, in the store, id is defined. This seems to be due to [this code](https://github.com/choojs/choo/blob/master/index.js#L78), here route handlers (views) are being called with a `eventName` and a single `data` parameter.

**edit**: I guess we could simply change this

```js
      var res = handler(self.state, function (eventName, data) {
        self.emitter.emit(eventName, data)
      })
```

to this

```js
      var res = handler(self.state, self.emitter.emit)
```

**Comment by heyitsmeuralex** [comment]
I'm almost certain this is intended - nanobus [should only support a single argument](https://github.com/choojs/nanobus#busemiteventname-data) 🤔

**Comment by YerkoPalma** [comment]
I'm not clear why it _should_ support a single argument. Also, it's not the way other event emitter work, e.g. [Node EventEmitter](https://nodejs.org/api/events.html#events_emitter_emit_eventname_args)

Maybe nanobus docs notation is unclear, it should be like 

### bus.emit(eventName, [...data])

**Comment by YerkoPalma** [comment]
Ahh @heyitsmeuralex saw your edit. It seems like docs should be clarified as nanobus does [parse all arguments from handlers](https://github.com/choojs/nanobus/blob/master/index.js#L19).

**Comment by yoshuawuyts** [comment]
I thought https://github.com/choojs/nanobus/pull/15 would've added support for multiple handlers. What's the error you're encountering?

**Comment by YerkoPalma** [comment]
nanobus does have support for multiple arguments. But the handler for `onroute` events, pass only the first argument to the views, so it is not an issue with nanobus code, it is an issue with choo code

**Comment by yoshuawuyts** [comment]
Fixed in `v6.6.1` :tada:
